.github/CODEOWNERS

```diff
@@ -936,6 +936,12 @@
 # PRLabel: %Communication - Resource Manager
 /sdk/communication/Azure.ResourceManager.Communication/            @archerzz @ArcturusZhang @ArthurMa1978
 
+# ServiceLabel: %Network - Front Door %Mgmt
+# ServiceOwners:                                                   @cdnfdsuppgithub
```

Comment:
The account `cdnfdsuppgithub` does not have the needed public organizations nor write permissions to the repository.   This needs to be resolved before it can be added as a code owner.